internal/ethapi/api.coreth_test.go

```diff
@@ -51,56 +65,84 @@ func TestSuggestPriceOptions(t *testing.T) {
 			name:             "minimum_values",
 			estimateBaseFee:  bigMinBaseFee,
 			suggestGasTipCap: bigMinGasTip,
+			cfg:              testCfg,
 			want: &PriceOptions{
 				Slow: newPrice(
 					minGasTip,
-					minBaseFee+minGasTip,
+					uint64(minBaseFee+minGasTip),
 				),
 				Normal: newPrice(
 					minGasTip,
-					minBaseFee+minGasTip,
+					uint64(minBaseFee+minGasTip),
 				),
 				Fast: newPrice(
 					minGasTip,
-					(fastFeeNumerator*minBaseFee)/feeDenominator+(fastFeeNumerator*minGasTip)/feeDenominator,
+					(testCfg.FastFeePercentage*uint64(minBaseFee)/feeDenominator)+(testCfg.FastFeePercentage*uint64(minGasTip)/feeDenominator),
```

Comment:
is it possible to set `fastFeeNumerator := testCfg.FastFeePercentage` and other local variables in this test to make the diff smaller?